src/tcpedit/plugins/dlt_en10mb/en10mb.c

```diff
@@ -586,22 +586,34 @@ dlt_en10mb_get_mac(tcpeditdlt_t *ctx, tcpeditdlt_mac_type_t mac, const u_char *p
 int
 dlt_en10mb_l2len(tcpeditdlt_t *ctx, const u_char *packet, const int pktlen)
 {
+    int l2len;
     struct tcpr_ethernet_hdr *eth = NULL;
     
     assert(ctx);
     assert(packet);
-    assert(pktlen);
+
```

Comment:
This mode is acceptable if done to all 13 plugins. Currently you have only modified Ethernet. 

Once this change is made, the PR looks good to me.